fi/development.html

```diff
@@ -0,0 +1,210 @@
+    <link href="http://fonts.googleapis.com/css?family=Open+Sans:300,400,700|Gentium+Basic" rel="stylesheet" type="text/css" />
+    <link rel="icon" href="/static/SymPy-Favicon.ico" />
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/2.0.2/jquery.min.js"></script>
+    <script type="text/javascript" src="/static/javascript/main.js"></script>
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <h1><a href="index.html"> <img src="/static/images/logo.png" alt="SymPy Logo" /> SymPy</a></h1>
+        <div id="mobile-menu"><i class="icon-ellipsis-vertical"></i></div>
+      </header>
+      <nav id="main-navigation" class="row">
+        
+        
+        <ul>
+          <li class=""><a href="index.html">Pääsivu</a></li>
+          <li class=""><a href="features.html">Ominaisuudet</a></li>
+          <li class=""><a href="download.html">Lataa</a></li>
+          <li class=""><a href="http://docs.sympy.org">Dokumentaatio <i class="icon-external-link-sign"></i></a></li>
+          <li class=""><a href="support.html">Tuki</a></li>
+          <li class="active"><a href="development.html">Kehitys</a></li>
+          <li class=""><a href="donate.html">Lahjoita</a></li>
+          <li><a href="http://live.sympy.org/">Online-komentokehoite <i class="icon-external-link-sign"></i></a></li>
+        </ul>
+        
+      </nav>
+      <section class="row" id="main">
+        <div class="col span_7 clr small_span_4">
+          
+          <article id="content">
+            
+
+<h2 class="title">Kehitys</h2>
+<div class="inner">
+  <p class="first">
+    Me olemme kiinnostuneita mistä tahansa avustuksesta. On olemassa monta tapaa jolla voit
+      avustaa:
+    <ul>
+      <li>
+        Mikäli kirjoitit jotakin kiinnostavaa SymPy-ohjelmistolla, ole hyvä ja lähetä meille patch
+         lähdekoodilla (ole hyvä ja katso lisätiedot alla).
+      </li>
+
+      <li>
+        Mikäli havaitsit bugin (tai haluat meidän tietävän mitä ajattelet), kerro meille
+         <a href="http://groups.google.com/group/sympy">postituslistalla</a>. Voit myös
+        lisätä bugiraportin (tai kommentin) tällä <a href="https://github.com/sympy/sympy/issues/">sivulla</a>.
+      </li>
+      <li>Voit blogata kuinka käytät SymPy-ohjelmistoa.</li>
+      <li>Voit avustaa meitä kirjoittamaan dokumentaatiota (lähettämällä meille patchin).</li>
+      <li>Voit avustaa ihmisiä <a href="http://groups.google.com/group/sympy">postituslistalla</a>.</li>
+      <li>Voit avustaa meitä tarkistamalla patcheja.</li>
+      <li>Voit lähettää meille patcheja joissa on bugikorjauksia tai uusia ominaisuuksia.</li>
+    </ul>
+  </p>
+  <p>
+    Paras tapa avustaa lahjoittamalla lähdekoodia tai dokumentaatiota on lähettämällä meille patchin.
+  </p>
+</div>
+
+<h2 class="title">Kuinka lähettää patchi</h2>
+<div class="inner">
+  <p class="first">
+    Paras tapa on lähettää meille pull request
+    <a href="http://github.com/sympy/sympy">sympy/sympy</a> repositoryyn.
```

Comment:
Sure, I'll translate these as soon as I can, probably tomorrow morning

Repository = säilytyspaikka? varasto? 
Pull request = vetopyyntö? 

Most of the sources I've checked are using English terms directly and there are no standard translations for these. If you have any recommendations, please let me know.